doc/source/WorkingWithTheNodeGraph/SpreadsheetNode/index.md

```diff
@@ -150,6 +150,19 @@ Once connected, the corresponding column will imitate the tweak plug's elements,
 
 The enabled switch is tied to the cell's enabled state: disabling the cell disables the tweak. A tweak plug cell can be disabled in the Default row, letting you bypass the tweak entirely when none of the rows apply.
 
+### Working with a spreadsheet ###
+
+The spreadsheet generally behaves like a traditional 'office' spreadsheet application:
+
+- Single-click a cell to select it.
+- Double-click a cell to edit it's value.
```

Comment:
it's -> its

Reply:
Thanks John, good spot. Sorry about that, fixed [here](https://github.com/GafferHQ/gaffer/pull/4068/commits/3cc504bcae498840688ccc59a4f71d75bb45a10d#diff-c00c5b8242d2870f16465bbeff8f1554e92353839563d5acee6c0855540b37d3R158).